ndt-server.go

```diff
@@ -117,9 +116,16 @@ func main() {
 
 	platformx.WarnIfNotFullySupported()
 
+	// Setup sequence of access control http.Handlers.
+	ac := alice.New()
 	tx, err := access.NewTxController(*maxRate)
-	rtx.Must(err, "Failed to allocate new TxController")
-	go tx.Watch(ctx)
+	if err == nil {
+		// Only watch interface and run tx.Limit on success.
+		go tx.Watch(ctx)
+		ac.Append(tx.Limit)
+	} else {
+		log.Println("WARNING: access.TxController disabled:", err)
```

Comment:
I am confused by why the access controller, when disabled, returns an error? Perhaps I just don't know enough about how it works, but I'd have found more understandable if it returned something like `tx.ErrDisabled`.

(Is there a scenario where we want to disable it? Maybe is that testing? Can we maybe use command line for that?)

Reply:
We have two flags currently for this feature, -txcontroller.device and -max-rate.

I've changed the default value for device to the empty string, requiring that it be specified to be enabled. What do you think?